snowpack/assets/require-or-import.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+const { pathToFileURL } = require('url');
+const NATIVE_REQUIRE = eval('require');
+const NATIVE_IMPORT = (filepath) => import(filepath);
+
+/**
+ * A utility function to use Node's native `require` or dynamic `import` to load CJS or ESM files
+ * @param {string} filepath 
```

Comment:
If this isn’t the perfect representation for the current state of JS I don’t know what is! 🤪 Good work at absorbing some pain in Snowpack for a better user experience. 💯

Reply:
Hahaha this is a really clever solution!